src/scala/org/pantsbuild/zinc/compiler/native-image-substitutions/Substitutions.java

```diff
@@ -0,0 +1,29 @@
+package scala.tools.nsc.substitutions;
```

Comment:
Can you add a comment at the top of this file (or in a README) explaining what it does, how these substitutions were reached at, and when people should add/remove things to this file?

Reply:
I'm not sure that we're going to be able to publish things in this package to maven-central.

If they need to live at this package location, we might need to dynamically add them to the image at native-image build time...?

Reply:
I'm 90% sure that the package names are arbitrary, this was ripped from the example at https://github.com/graalvm/graalvm-demos/tree/master/scala-days-2018/scalac-native and this package doesn't actually exist in scalac.

Reply:
Also, I intend to add very clear documentation on how every build or runtime failure was investigated and worked around before I mark this un-WIP.

Reply:
I have added a README.md describing where these substitutions came from and how they are used.

Reply:
I have also modified the package names to be nested in the pantsbuild package namespace.